scipy/stats/_morestats.py

```diff
@@ -3373,6 +3377,178 @@ def func(x):
+    consistent application. She wants to use the mist blower with the
+    smaller variance, which means more consistent application.
+    She wants to test that the variance of Type A (0.087 gal.2) is
+    significantly greater than the variance of Type B (0.073 gal.2)
+    using a level of significance of 0.05.
+
+    Type A has a sample of 16 mist blowers with a variance of 0.079.
+    Type B has a sample of 21 mist blowers with a variance of 0.060.
+
+    >>> import numpy as np
+    >>> typeA = np.array([
+    ...     11.2, 11, 11.3, 11.5, 11, 11, 11.5, 11.3, 10.9,
+    ...     11.2, 10.9, 10.9, 11.1, 10.5, 10.6, 10.9
+    ... ])
+    >>> typeB = np.array([
+    ...     10.8, 11.2, 10.9, 10.7, 11.5, 11, 11.1, 10.7, 11, 11.1, 10.8,
+    ...     11.2, 10.9, 11, 10.9, 11.6, 11.1, 10.8, 11.3, 10.8, 11.2
+    ... ])
+
+    >>> from scipy.stats._morestats import f_equality_test
+    >>> args = [typeA, typeB]
+    >>> res = f_equality_test(*args, alternative='greater')
+    >>> res.statistic
+    1.3079968329374523
+    >>> res.pvalue
+    0.7169386351928994
+
+    The test statistic is not larger than the critical value (it does not
+    fall in the rejection zone) so we fail to reject the null hypothesis.
+    While the variance of Type B is mathematically smaller than the variance
+    of Type A, it is not statistically smaller. There is not enough statistical
+    evidence to support the claim that the variance of Type A is significantly
+    greater than the variance of Type B. Both mist blowers will deliver the
+    chemical with equal consistency.
+
+    """
+    k = len(samples)
+    if k != 2:
+        raise ValueError("Must enter two input sample vectors.")
+    
+    if alternative not in {'two-sided', 'greater', 'less'}:
+        raise ValueError("'alternative' must be 'two-sided',"
+                         " 'greater', or 'less'.")
+    
+    # Handle empty input and input that is not 1d
+    for sample in samples:
+        if np.asanyarray(sample).size == 0:
+            NaN = _get_nan(*samples)  # get NaN of result_dtype of all samples
+            return FResult(NaN, NaN)
+
+    Ni = np.empty(k)
+    ssq = np.empty(k, 'd')
+    for j in range(k):
+        Ni[j] = len(samples[j])
+        ssq[j] = np.var(samples[j], ddof=1)
+
+    if ssq[0] > ssq[1]:
+        larger_variance = ssq[0]
+        smaller_variance = ssq[1]
+        df1 = Ni[0] - 1
+        df2 = Ni[1] - 1
+    else:
+        larger_variance = ssq[1]
+        smaller_variance = ssq[0]
+        df1 = Ni[1] - 1
+        df2 = Ni[0] - 1
+         
+    # Calculate the F-statistic
+    f_statistic = larger_variance / smaller_variance
+
+    # Calculate the p-value using the F-distribution
+    if alternative == 'two-sided':
+        p_value = 2.0 * distributions.f.sf(f_statistic, df1, df2)
+    elif alternative == 'greater':
+        p_value = stats.f.cdf(f_statistic, df1, df2)
+    else: 
+        p_value = distributions.f.sf(f_statistic, df1, df2)
```

Comment:
```suggestion
    if alternative == 'two-sided':
        p_value = 2.0 * distributions.f.sf(f_statistic, df1, df2)
    elif alternative == 'greater':
        p_value = stats.f.cdf(f_statistic, df1, df2)
    else: 
        p_value = distributions.f.sf(f_statistic, df1, df2)
```
Please use `scipy.stats._stats_py._get_pvalue`